mypy/test/data.py

```diff
@@ -379,7 +379,7 @@ def expand_errors(input: List[str], output: List[str], fnam: str) -> None:
         # The first in the split things isn't a comment
         for possible_err_comment in input[i].split('#')[1:]:
             m = re.search(
-                '^([ENW]):((?P<col>\d+):)? (?P<message>.*)$',
+                '^([ENW]):((?P<col>\d+):)?((?P<backtrack>-\d+):)? (?P<message>.*)$',
```

Comment:
I would prefer this as a separate PR, proposed syntax is debatable and could only delay this PR.